webrtc-stats.html

```diff
@@ -3957,17 +3222,6 @@ <h2>
                   on the <code class=ice>"transport"</code> defined in [[!RFC5245]] section 15.1.
                 </p>
               </dd>
-              <dt>
-                <dfn>relayProtocol</dfn> of type <span class=
```

Comment:
Revive this (relayProtocol), it was only removed from the definition section not the dictionary

Reply:
Same with url